changelog.d/7227.misc

```diff
@@ -0,0 +1 @@
+Update the Debian installation to include information about .well-known in addition to DNS SRV.
```

Comment:
in theory we don't mandate changelog.d files for stuff that only affects the debian package, fwiw. It's harmless enough though.